gap/FinSetsForCAP.gi

```diff
@@ -1005,8 +1007,17 @@ AddClassifyingMorphismOfSubobjectWithGivenSubobjectClassifier( FinSets,
       return MapOfFinSets( range, r, omega );
 end );
 
+SetRangeCategoryOfHomomorphismStructure( FinSets, FinSets );
+
 ##
-Finalize( FinSets );
+AddHomomorphismStructureOnObjects( FinSets,
+  ExponentialOnObjects );
+
+##
+AddHomomorphismStructureOnMorphismsWithGivenObjects( FinSets,
+  ExponentialOnMorphismsWithGivenExponentials );
```

Comment:
Does it make sense to implement this partially? Without `Interpret*` I think nobody will be able to use this, and even worse, it prevents your derivation of the homomorphism structure from the closed structure and a bidirectional interpretation to kick in in this case.

Reply:
No, this was not intended to be part of this commit.